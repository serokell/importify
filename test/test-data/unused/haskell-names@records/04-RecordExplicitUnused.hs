module RecordExplicitUnused where

import           Language.Haskell.Names (Symbol (Value, symbolModule, symbolName))

foo = Value {symbolModule = undefined}
