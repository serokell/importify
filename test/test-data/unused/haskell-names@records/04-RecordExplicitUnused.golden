module RecordExplicitUnused where

import           Language.Haskell.Names (Symbol (Value, symbolModule)            )

foo = Value {symbolModule = undefined}
